deploy/role.yaml

```diff
@@ -55,6 +55,30 @@ rules:
   - deployments/finalizers
   verbs:
   - update
+- apiGroups:
+  - ""
+  resources:
+  - services    # Services is the way we provide metrics for the operator
+  - services/finalizers
```

Comment:
Mostly looks good, but I wonder if you based this on some docs? I compared these rules with what jaeger operator uses and they don't grant services/finalizers at all.
Note that I haven't tried just removing the line, let me do this now, I'm interested in docs either way :)

Reply:
Replying to myself, this is needed, otherwise you still get the message with the finalizers. Weird that other operators don't have that.